README.md

```diff
@@ -105,7 +105,6 @@ To setup and install this sample Recipe-App project, follow the below steps:
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍